querybook/webapp/lib/templated-query/validation.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * This is a super basic function to check whether or not a string
+ * uses jinja templating, it should only be used as a safety measure
+ * since it does not provide perfect correctness
+ *
+ * @param query
+ * @returns boolean
+ */
+export function isQueryUsingTemplating(query: string): boolean {
```

Comment:
so it wont cover the case of template strings are in the query comments?

Reply:
tetmplate strings in query comments are still rendered, and if it is multi line, it would impact the query still, so there is no good way to prevent that